build-system/global-configs/prod-config.json

```diff
@@ -37,6 +38,7 @@
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 0,
   "ios-scrollable-iframe": 0,
+  "macro-after-long-task": 1,
```

Comment:
this will completely launch the feature in prod. I though you only want URL opt-in?

Reply:
You're absolutely right, fixed with 52fdceb.